frontend/src/globals/unity-util.ts

```diff
@@ -16,10 +16,14 @@
  */
 
 /* eslint-disable no-underscore-dangle */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
🤔  bit confused with this one, your next line is a comment?

Reply:
Me as well I think that may have been a merge artefact!